druid-shell/src/platform/windows/window.rs

```diff
@@ -1277,35 +1532,140 @@ impl WindowHandle {
         }
     }
 
-    // TODO: Implement this
-    pub fn show_titlebar(&self, _show_titlebar: bool) {}
+    pub fn show_titlebar(&self, show_titlebar: bool) {
+        if let Some(w) = self.state.upgrade() {
+            w.has_titlebar.set(show_titlebar);
+            if let Ok(mut q) = w.blocked_queue.try_borrow_mut() {
```

Comment:
Maybe consider a newtype for `BlockedQueue` (or "DeferredQueue" to use my preferred name), and have a method to handle all this boilerplate.